check_modulemd.py

```diff
@@ -24,16 +24,28 @@ def setUp(self):
         Verify required modulemd file parameter has been specified, exists,
         and can be loaded. The file name and loaded metadata are saved.
         """
+        mmd = modulemd.ModuleMetadata()
         mdfile = self.params.get('modulemd')
+        # try to use module testing farmework if possible
+        # https://pagure.io/modularity-testing-framework
+        try:
+            mtf_backend = module_framework.CommonFunctions()
+            tmdfile = "temp_modulemdfile.yaml"
```

Comment:
I would prefer to use python temporary function like https://docs.python.org/2.7/library/tempfile.html#module-tempfile